examples/b2b/checkout/retrigger-place-order/classes/RetriggerPlaceOrderAction.cls

```diff
@@ -0,0 +1,26 @@
+global class RetriggerPlaceOrderAction {
+    
+    @InvocableMethod(callout=false label='Create Order and order summary for Cart' description='Create order and order summary' category='Commerce')
+    global static List<String> retriggerPlaceOrder(List<ID> cartOrorderIds) {
+        if (cartOrorderIds == null || cartOrorderIds.size() != 1) {
```

Comment:
I can remove that check
The only reason I kept is since the invocable method can only have List<> as input, its always better to check the list size

Reply:
Since the apex invocable method interface is a list of objects, I am wondering if user can create a custom flow which may pass in a collection of Ids.

Reply:
The RetriggerPlace Flow invokes this action and it only has one OrderOrCartId, added a comment based on the discussion